lab_3_generate_by_ngrams/main.py

```diff
@@ -346,7 +515,13 @@ def prune_sequence_candidates(
 
         In case of corrupt input arguments return None.
         """
-
+        if not(isinstance(sequence_candidates, dict)
+               and sequence_candidates):
+            return None
+        sorted_sequence_candidates = dict(sorted(sequence_candidates.items(), key=lambda x: x[1]))
```

Comment:
Please don't sort a dictionary, the order is not guaranteed

Reply:
Fixed